EventGenerator/src/RPCGun_module.cc

```diff
@@ -54,12 +55,12 @@ namespace mu2e {
     struct Config {
       using Name=fhicl::Name;
       using Comment=fhicl::Comment;
-        fhicl::Atom<art::InputTag> inputSimParticles{Name("inputSimParticles"),Comment("A SimParticleCollection with input stopped muons.")};
-        fhicl::Atom<unsigned> verbosity{Name("verbosity"),0};
+        fhicl::Atom<art::InputTag> inputSimParticles{Name("inputSimParticles"),Comment("A SimParticleCollection with input stopped pions")};
+        fhicl::Atom<unsigned> verbosity{Name("verbosity"),Comment("Add additional printout"), 0};
         fhicl::Atom<std::string> RPCType{Name("RPCType"),Comment("a process code, should be either RPCInternal or RPCExternal") };
         fhicl::DelegatedParameter spectrum{Name("spectrum"), Comment("Parameters for BinnedSpectrum")};
-        fhicl::Atom<bool> pionDecayOff{Name("pionDecayOff"),true};
-        fhicl::Atom<bool> doHistograms{Name("doHistograms"),false};
+        fhicl::Atom<bool> pionDecayOff{Name("pionDecayOff"),Comment("Assume pion decay was turned off, produce event weights"), true};
```

Comment:
This comment makes me realize we have no way to definitively flag pions produced with infinite lifetime. Outside the scope of this PR we should add a data product for that.